test/e2e/test_gcp_broker.go

```diff
@@ -177,5 +177,8 @@ func createGCPBroker(client *lib.Client) (url.URL, string) {
 	if err != nil {
 		client.T.Error(err.Error())
 	}
+	// Avoid propagation delay between the controller reconciles the broker config and
+	// the config being pushed to the configmap volume in the ingress pod.
+	time.Sleep(60 * time.Second)
```

Comment:
We really take that long?

Reply:
Just think if the forcing update configmap function failed to update, it may take 1 min to sync (for the worst case). But it is true that it is too long. I changed to 15 second in  #1415. 

Also, I add a new sender pod which have retry for 404 and 503 error in #1415 (after sync with Ian)